doc/example-config

```diff
@@ -1018,15 +1007,26 @@
 #
 # prepopulate-query-feeds yes
 
-####  ssl-verify
+####  ssl-verifyhost
+#
+# If no, skip to verify the certificate's name against host.
```

Comment:
This should say "skip verification of the certificate's name against host". For details, ee my comment for `ssl-verifypeer`.